packages/cli/src/CommandsLogic.ts

```diff
@@ -73,7 +77,10 @@ export default class CommandsLogic {
     deployment: GSNContractsDeployment,
     mnemonic?: string
   ) {
-    let provider: HttpProvider | HDWalletProvider = new Web3.providers.HttpProvider(host)
+    let provider: HttpProvider | HDWalletProvider = new Web3.providers.HttpProvider(host, {
+      keepAlive: true,
```

Comment:
thought keepAlive was a default... 
if it isn't, then its important to add - but to real server, not the command-line tool.

Reply:
debugger was disconnecting. not sure if this helped in fact